.github/workflows/acceptance-tests.yml

```
@@ -0,0 +1,66 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+name: "AWS Acceptance Tests"
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "A branch or SHA"
+        default: 'main'
+        required: false
+      run_pattern:
+        description: "The pattern to run tests on"
+        default: "*"
```

Using `"*"` is an invalid regexp expression for go test
```suggestion
        default: "TestAcc"
```

---

Good call, the pattern was borked indeed, I initially shot for `.*`, but `TestAcc` does the trick indeed.